### ISSUE TITLE
Usage of "master" in MasterPackageList repository name

### ISSUE THREAD

**Issue by a2** [SwiftPackageIndex/SwiftPackageIndex-Server#408]
The greater technology community has voiced concerns around Git’s use of "master" as the default branch name (among other similarly problematic words). I wanted to bring attention to the usage of "master" in the repository name for [MasterPackageList](https://github.com/SwiftPackageIndex/MasterPackageList) as well. Therefore, I created this issue to start a dialogue on the matter. Phrasing like "package list index" or "package list source" might be less harmful. I'm happy to discuss this further. Thank you for your consideration.

**Comment by daveverwer** [comment]
Ohhh sorry I misunderstood from our Twitter conversation about this just now. I thought you were talking about the `master` branch name, I hadn't even noticed it in the repo name. 😬

I agree that we should do it, but the change to the repo is a little tricky as it involves changing and deploying new server software too. We'll do it, but it will take a bit of coordination with deployment.

We should also change the default branch name from `master` to `main`, just like we did on this project repo. However, before we do I would love to know what happens to forks. The package list repo has 214 forks open right now. Some of those are stale, and won't ever be merged back, but some are current. Does anyone know what happens to a fork when the default branch name on the remote changes?

**Comment by a2** [comment]
As far as I know, which may be completely wrong, but the concept of a fork is a one-time mirroring of the repo at the time of forking, with some additional metadata on the GitHub backend that shows that the repo came from this one. I believe that developers would have to refetch the main branch on the upstream repo anyway?

```bash
git add upstream https://github.com/SwiftPackageIndex/NewRepoName.git
git fetch upstream
git switch -c my_branch_name upstream/main
```

**Comment by daveverwer** [comment]
That's good, the biggest issue will be communicating that to the people who have forked it. Maybe a large banner across the top of the README.

However, I'm a little reluctant to divert from the main message of the README. While there are a lot of forks, they are easy to make and the changes to the JSON file are trivial. Maybe we just hope people re-fork if they are struggling to merge back in.

**Comment by finestructure** [comment]
TBH, I don't think it should be huge problem for forks. It should be clear what the upstream is from the fact that there's a single branch called `main` and even if you continue on your own old fork with `master` all you'll do is choose this new one as a merge target.

And if you want to update your local copy I think that's also fairly clear.

We could just see how it goes and add a little FAQ section to the README in case people struggle?

**Comment by daveverwer** [comment]
OK, @finestructure  and I worked on this this morning and it's all fixed and deployed via #413. Thanks for bringing it up. 

The package list repository is now called `PackageList` and has a `main` default branch.
